.gitignore

```
@@ -125,4 +125,6 @@ modules.xml
 .idea/sonarlint
 
 # End of https://www.toptal.com/developers/gitignore/api/clion+all,c++
-IntelliJ
\ No newline at end of file
+IntelliJ
+
+cmake-build-debug
```

end the file with newline
```suggestion
cmake-build-debug

```